### PR TITLE
RDM-6193-cleaning ccd flux config

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: k8s/demo/common/ccd/latest-ccd-chart.yaml [hunk 1]
```diff
@@ -63,40 +55,13 @@ spec:
             secretRef: ccd-admin-web-oauth2-client-secret
             key: key
         environment:
-#         TODO move this to ccd-chart
-          ADMINWEB_LOGIN_URL: https://idam-web-public.demo.platform.hmcts.net/login
           ADMINWEB_ROLES_WHITELIST: 'ccd-import,ccd-import-validate'
```
Comment (mario-paniccia) [inline k8s/demo/common/ccd/latest-ccd-chart.yaml]: why this one has been left here?

File: k8s/demo/common/ccd/latest-ccd-chart.yaml [hunk 2]
```diff
@@ -63,40 +55,13 @@ spec:
...
-#          TODO update IDAM_BASE_URL, IDAM_OAUTH2_TOKEN_ENDPOINT, IDAM_OAUTH2_LOGOUT_ENDPOINT in ccd-chart and remove it from here
-          IDAM_BASE_URL: '{{ .Values.global.idamApiUrl }}'
-          IDAM_OAUTH2_TOKEN_ENDPOINT: '{{ .Values.global.idamApiUrl }}/oauth2/token'
-          IDAM_OAUTH2_LOGOUT_ENDPOINT: '{{ .Values.global.idamApiUrl }}/session/:token'
           IDAM_ADMIN_WEB_SERVICE_KEY: AAAAAAAAAAAAAAAA
```
Comment (mario-paniccia) [inline k8s/demo/common/ccd/latest-ccd-chart.yaml]: why this one has been left here?

File: k8s/demo/common/ccd/latest-ccd-chart.yaml (hunk 1 above)
Comment (akdasari) [reply k8s/demo/common/ccd/latest-ccd-chart.yaml]: moved to chart-ccd

File: k8s/demo/common/ccd/latest-ccd-chart.yaml (hunk 2 above)
Comment (akdasari) [reply k8s/demo/common/ccd/latest-ccd-chart.yaml]: moved to chart-ccd

**Pull request closed without merging**